changelog.md

```diff
@@ -1,10 +1,24 @@
 
 # Change Log
 All notable changes to this project will be documented in this file.
- 
+
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
- 
+
+## [0.11.0] - 2019-10-11
+
+### Added
+
+
+
+### Changed
```

Comment:
I think you can drop the Added/Changed, since we'll be pushing another version here pretty soon ;)